docs/src/Sheet.doc.js

```diff
@@ -1,17 +1,24 @@
 // @flow strict
 import type { Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
-import Card from './components/Card.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
 card(
   <PageHeader
     name="Sheet"
-    description="Sheets are side surfaces that allow users to view information or complete sub-tasks in a workflow while keeping the context of the current page."
+    description="Sheets are surfaces that allow users to view optional information or complete sub-tasks in a workflow while keeping the context of the current page. The most common example of Sheet displays content in a panel that opens from the side of the screen for the user to read or input information. Sheets have a default internal padding for content."
```

Comment:
nittiest of nits: "Sheets have default internal padding…" The "a" in there sounds kinda weird?